tests/bdd/steps/lms_db.py

```diff
@@ -20,13 +20,11 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.engine = sqlalchemy.create_engine(TEST_DATABASE_URL)
         self.session_maker = sessionmaker(bind=self.engine.connect())
-        db.init(self.engine, stamp=False)
+        db.init(self.engine, stamp=False, drop=True)
```

Comment:
This I thinks it's just easier, now we do
`db.init(drop=True)` which drops any tables that might have existed, creates them again.

We don't need `_initial_wipe` anymore.